live-examples/js-examples/expressions-commaoperators.html

```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">var x = 1;
+
+console.log(x++, x);
+// expected output: 1
+
+console.log(x);
+// expected output: 2
+
+x = (y = 2, z = 3);
```

Comment:
Nit: I'm not sure why this can't just be `x=(2, 3)`.

Reply:
I guess it was to show that you can do things within the brackets. Have revised so that we have something like this:

```
var x = 1;

x = (x++, x);

console.log(x);
// expected output: 2

x = (2, 3);

console.log(x);
// expected output: 3
```